src/preset_cli/cli/superset/sync/dbt/command.py

```diff
@@ -75,6 +81,7 @@ def dbt_core(  # pylint: disable=too-many-arguments, too-many-locals
     import_db: bool = False,
     disallow_edits: bool = True,
     external_url_prefix: str = "",
+    exposures_only: bool = True,
```

Comment:
since you have this defaulting to False above, do you want the default here to match?

Reply:
Oh, good catch!

Reply:
It doesn't really matter, but it's good to be consistent. Updated it.